frontend/src/components/groupusertable/GroupUserTableItem.tsx

```diff
@@ -82,7 +82,7 @@ export const GroupUserTableItem = ({ groupUser, groupData, punishmentTypes, data
     <AccordionItem value={groupUser.user_id}>
       <AccordionTrigger className="relative flex cursor-pointer justify-between gap-x-6 py-5 rounded-lg md:rounded-xl hover:bg-gray-50">
         <div className="flex items-center gap-x-2">
-          <span className="flex h-8 w-8 md:h-12 md:w-12 pt-1 items-center justify-center rounded-full bg-indigo-100 dark:bg-gray-800 text-lg md:text-3xl text-[#4C4C51] relative">
+          <span className="flex h-8 w-8 md:h-12 md:w-12 items-center justify-center rounded-full bg-indigo-100 dark:bg-gray-800 text-lg md:text-3xl text-[#4C4C51] relative">
```

Comment:
Are you sure we should be making this big of changes right before launch? 🤔

Reply:
Yes